app/views/users/_profile_header.html.erb

```diff
@@ -24,7 +24,11 @@
         </span>
       </h1>
       <p class="profile-description" itemprop="description">
-        <%= @user.summary.presence || ["404 bio not found"].sample %>
+        <% if @user.has_attribute?(:summary_html) %>
```

Comment:
I don't think this check is needed, summary html will be present for all users, you just need to check if it has data in it or not.

Reply:
It looks like this is used in https://github.com/thepracticaldev/dev.to/blob/9dd4ec6bafbd33970df06f10fdcbf5e54ec0fd65/app/views/organizations/show.html.erb#L6